### PR TITLE
Contributed to 4 program

### PR DESCRIPTION
**Pull request by Zannatul-Naim**

### CODE REVIEW

File: A-set/199A.Hexadecimals_theorrem.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#include<bits/stdc++.h>
+#include<sstream>
+using namespace std;
```
Comment (Waqar-107) [inline A-set/199A.Hexadecimals_theorrem.cpp:3]: Add an extra blank line before this. Do this for the rest of the 4 solutions.

**Review by Waqar-107 — CHANGES_REQUESTED** [review]
Can you make some changes?

1. The naming of the files doesn't seems right. Check the contributions guideline. Just copy the name from codeforces and paste. Like `199A. Hexadecimals Theorem`
2. The solution for `199A` already exists. So, please remove it. Kepp the rest.
3. Some lines of code have 4-spaces as indentation and some have 2-spaces. Please stick to only one.

File: A-set/199A.Hexadecimals_theorrem.cpp (hunk 1 above)
Comment (Zannatul-Naim) [reply A-set/199A.Hexadecimals_theorrem.cpp]: Updated. Please check

**Review by Waqar-107 — APPROVED** [review]
Looks good 🚀

**Comment by Zannatul-Naim** [comment]
Thanks

**Pull request merged**